client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -22,34 +22,45 @@ const ChecklistItem = ( { task }: { task: Task } ) => {
 				'not-completed': ! isCompleted && ! keepActive, // a task that hasn't been completed yet
 			} ) }
 		>
-			<Button
-				className="launchpad__checklist-item"
-				disabled={ taskDisabled }
-				data-task={ id }
-				onClick={ actionDispatch }
-			>
-				{ isCompleted && (
-					// show checkmark for completed tasks regardless if they are disabled or kept active
-					<div className="launchpad__checklist-item-checkmark-container">
+			{ id === 'first_post_published' || id === 'link_in_bio_launched' ? (
```

Comment:
Instead of handling this differently via a hardcoded list of IDs, I wonder if it would make more sense to handle this with a prop to denote it a primary/cta, and handle setting that from the checklist component's array map function or something similar?

It seems like its always the last checklist item that will always be the CTA button, so it seems like we could avoid looking at IDs with updating the [map callback](https://github.com/Automattic/wp-calypso/blob/be01f4a57ec2a786f875a22214e199059155dcd5/client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist.tsx#L11) to do something like:
```
return <ChecklistItem isCTA={ is last item of the map } ...the other props />
```

Then this checklist-item component only has to care about the boolean prop, and we can avoid checking a whitelist of hardcoded ids.